js/background.js

```diff
@@ -1183,15 +1227,17 @@ get_l10n_data = (SAFARI ? chrome.i18n._getL10nData : undefined);
 
 // BROWSER ACTION AND CONTEXT MENU UPDATES
 (function() {
-    if (SAFARI)
+    if (SAFARI) {
         return;
+    }
```

Comment:
@kpeckett It's better readable for me like it is, because it's more obvious on the first moment that the following code shouldn't be running in Safari.
Replacing ```if (SAFARI)``` with ```if (!SAFARI)``` would also cause another code nesting, because the entire function would need to be wrapped in another ```if``` block.